react-sizeme.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module 'react-sizeme' {
```

Comment:
Actually, you don't need this, since your `package.json` includes `types` property.
I doubt that this will affect end-users somehow, but `declare module` syntax skipped in both `DefinetlyTyped` typings and generated by TypeScript compiler.
`declare module` used only for augmenting host by plugin (e.g. some packages for `express` do use `declare module 'express'` for best developers experience) or for typing untyped package from your actual code, when there are no typings for it neither in package nor `@types/`

Reply:
Thanks for reviewing, I'll remove that. I went to see some typings generated by the TS compiler and it emits `export declare class Foo` instead of just `export class Foo`. Should I add the `declare` as well? Thanks!

Reply:
I don't think so, I don't know why actually TS adds `declare` to declarations, but it doesn't seem to affect compilation.
But this affects readability of definitions (at least in VS Code), because breaks  colorizing(I guess something is wrong with grammar when there are too many reserved words in one statement).

Also, I've just realized, that you don't have `react` import and you should.
I suggest to use `import { Component, ComponentType } from 'react'`, so typings will work with both `esModuleInterop` set `true` and to `false`.